requirements.txt

```diff
@@ -8,7 +8,6 @@ static3==0.5.1
 wsgiref==0.1.2
 -e git+https://github.com/funkybob/django-flatblocks.git#egg=django-flatblocks
 sendgrid-django
-psycopg2==2.5.4
```

Comment:
@GoFroggyRun if the main issue that you are having issues using `postgres` locally? If thats the case, there is the option of using SQLite for local development. It's what i use, since I have to alter the database frequently.

Is the problem that your `psycopg2` version is causing problems, so you had to upgrade it? In that case @martinholmer that's a reasonable fix.